src/app/recipes/recipe-list/recipe-list.component.ts

```diff
@@ -17,9 +22,18 @@ export class RecipeListComponent implements OnInit {
                       5
                     );
 
-  constructor() { }
+  constructor(private apiClient: ApiService) { }
 
   ngOnInit() {
+    this.apiClient.signin('arol','Bananas')
+    .then(()=>this.apiClient.getRecipes())
```

Comment:
This is just an example how to implement the apiClient in a component.... 
`this.apiClient.getRecipes())` show have a parameter to work...
`console.log(error);` at line 35 also should be `console.log(err);`